### PR TITLE
LMB-519 | Handle subject to check later

### PR DESCRIPTION
**Pull request by JonasVanHoof**
## Description

Now that we check the incoming triples from our delta we want to update the logic check the subjects that are incoming but do not have all values set yet. We will do this by keeping the mandataris subjects in a queue that we still have to check.

## How to test

Logic should be updating inserting mandatarissen from the mandatees-decisions stream.
Everytime the endpoint is triggered and there are manually queued items it puts them in the current queue to be executed. As the queue is executing the queue executes the itens one by one so  I changed the parameter in the handle method only one subject instead of an array of subjects. 

## Links to other PR's

- https://github.com/lblod/mandataris-service/pull/17

### CODE REVIEW

**Comment by JonasVanHoof** [comment]
Log example:

```bash
mandataris-1  | |> Currently 14 items in manual queue.
mandataris-1  | |> Remaining number of tasks 72
mandataris-1  | |> Done with action for queue item
mandataris-1  | |> Handle Triples For Mandataris Subject (http://data.lblod.info/id/mandatarissen/1d91651ad2865f011442a693e02ae4c00d403e768bd16bb52e7f2b8e285bd023)
mandataris-1  | |> Mandataris uri: http://data.lblod.info/id/mandatarissen/1d91651ad2865f011442a693e02ae4c00d403e768bd16bb52e7f2b8e285bd023
mandataris-1  | |> Mandataris exists in LMB database? false
mandataris-1  | |> Mandaat for mandataris {
mandataris-1  | |> mandataris graph: http://mu.semte.ch/graph/organizations/11228df4ed78cc6b5c688c561570fa3b2a609540db8482a2866a8ed0cc398b08/LoketLB-mandaatGebruiker.
mandataris-1  | |> Found 2 in the staging graph for mandataris.
mandataris-1  | |> Persoon from mandataris: undefined.
mandataris-1  | |> Persoon does not exist: TODO in LMB-520
mandataris-1  | |> End of logic for mandataris subject: http://data.lblod.info/id/mandatarissen/1d91651ad2865f011442a693e02ae4c00d403e768bd16bb52e7f2b8e285bd023 
```

File: routes/mandatees-decisions.ts [hunk 1]
```diff
@@ -3,15 +3,29 @@ import Router from 'express-promise-router';
...
+    .map((changeset: Changeset) => changeset.inserts)
+    .flat();
+  const mandatarisSubjects = await getSubjectsOfType(
+    TERM_MANDATARIS_TYPE,
+    insertTriples,
+  );
+
+  mandatarisQueue.setMethodToExecute(handleTriplesForMandatarisSubject);
+  mandatarisQueue.addToQueue(mandatarisSubjects);
+  mandatarisQueue.moveManualQueueToQueue();
```
Comment (bfidlers) [inline routes/mandatees-decisions.ts:28]: I'm a bit confused about the manual queue. Am I right to assume you use the manual queue, so you only allow the queue to process these mandataris subjects if new triples come in?

File: data-access/mandatees-decisions.ts [hunk 2]
```diff
@@ -9,7 +9,10 @@ import {
 import { TERM_TYPE, sparqlEscapeTermValue } from '../util/sparql-escape';
 import { MANDATARIS_STATUS } from '../util/constants';
 
-const STAGING_GRAPH = 'http://mu.semte.ch/graphs/besluiten-consumed';
+export const TERM_STAGING_GRAPH = {
```
Comment (bfidlers) [inline data-access/mandatees-decisions.ts:12]: I don't really get the use of this type in the TERM_STAGING_GRAPH

Comment (bfidlers) [reply data-access/mandatees-decisions.ts:12]: Ah never mind, it is just so you can use the more general sparqlExcapeTermValue, right?

**Review by bfidlers — APPROVED** [review]
Haven't yet dared to run it, might be a good idea to try this together on Thursday or Friday. 

But this certainly looks good, and you can merge it already.

**Pull request merged**